app/components/DosesDescriptionList/DosesDescriptionList.tsx

```diff
@@ -19,20 +20,21 @@ export const DosesDescription = ({
   dosesPercent,
   children = null,
 }: DosesDescriptionProps) => (
-  <div className={styles["description-definition"]}>
-    <dt className={cx(styles["term"], "space-x-xs")}>
+  <div className="width-half">
+    <dt className={cx(styles["dose-label"], "margin-bottom-2xs")}>{term}</dt>
+    <dd className="flex flex-row justify-content-start align-items-center space-x-xs">
       <div
-        className={cx("data-text", { ["data-text-complete"]: hasMetTarget })}
+        className={cx(styles["percentage-value"], {
+          [styles["percentage-value-complete"]]: hasMetTarget,
+        })}
       >
-        {term}
```

Comment:
Term not being green anymore? 😢

Reply:
See "top right" of the figma file

![image](https://user-images.githubusercontent.com/22784983/140027648-61a24d32-12b7-4bab-8285-6f04391996cd.png)

Reply:
Ah cool ty!